momaland/envs/connect4/connect4_check.py

```diff
@@ -0,0 +1,49 @@
+"""MO Connect4 check."""
```

Comment:
I actually very much like these "check" files you have implemented. I think it would be good to have one of these sanity checks for each environment, maybe in a dedicated test runner? This should be discussed also with @rradules

Reply:
I agree. CrazyRL environments have these as well in the main environment files 👍

Reply:
Yes, agreed.